go-controller/pkg/cni/helper_linux.go

```diff
@@ -252,81 +254,100 @@ func setupInterface(netns ns.NetNS, containerID, ifName string, ifInfo *PodInter
 	return hostIface, contIface, nil
 }
 
+var chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890-"
+
+func generateId(length int) string {
```

Comment:
This function seems generic and can be reused for other purposes, I think it can be placed in some "util" package and make it public.